src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterArrayDeclarations2.java

```diff
@@ -9,28 +9,28 @@
 
     public class A {
         public int[][] create(int i, int j) {
-            return new int[3] [3];//incorrect, 12:30 
```

Comment:
We are expecting very few changes in inputs.
Only allowed changes are package and class name changes. No other changes.

Please revert these superfluous changes.
Please manually confirm there are no changes like this for other inputs as we can't see differences for new input files.